proto/storage/product_usage.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+option go_package = "storage";
+option java_package = "io.stackrox.proto.storage";
+
+import "google/protobuf/timestamp.proto";
+import "gogoproto/gogo.proto";
+
+package storage;
+
+// SecuredUnits represents a record of an aggregated secured clusters usage
+// metrics. The metrics are aggregated periodically, and put into the database.
+message SecuredUnits {
+    // id is not used to retrieve data, but serves mostly for compatibility with
+    // the current implementation of the query generator.
+    string id                           = 1 [(gogoproto.moretags) = 'sql:"pk,type(uuid)"'];
+    // timestamp stores the moment at which the values of the metrics below are
+    // aggregated.
+    google.protobuf.Timestamp timestamp = 2;
+    // num_nodes is the maximum number of secured nodes, observed across all
+    // registered clusters during last aggregation period.
+    int64 num_nodes                     = 3;
```

Comment:
unsigned?

Reply:
Let's not for consistency: `uint64` is a seldom guest in our proto files; a quick grepping shows `int64` is common for counts:
<img width="1070" alt="Screenshot 2023-08-23 at 09 58 40" src="https://github.com/stackrox/stackrox/assets/2613999/26eb94f9-466e-4c19-aeca-f5ce4622702a">

Reply:
Not a big issue...with the right validation `int64` because we do not expect counts to be negative.

Reply:
But we don't care much if they're negative.